src/events/eventBus/EventBusConfig.ts

```diff
@@ -59,4 +74,30 @@ const payloadBuilders = {
       scheduledDate: toUtcDateString(data.scheduledCorpusItem.scheduledDate),
     };
   },
+  approvedItemEvent(
+    eventType: string,
+    data: ReviewedCorpusItemPayload
+  ): ApprovedItemEventBusPayload {
+    // The nullish coalesce and checking for properties are due to
+    // union type in ReviewedCorpusItemPayload
+    const item = data.reviewedCorpusItem;
+    return {
+      eventType: eventType,
+      approvedItemId: item.externalId,
+      url: item.url,
+      title: item.title ?? undefined,
```

Comment:
we have to check them for undefined coz the rejectedItems won't have these field set?

Reply:
yep